README.md

```diff
@@ -63,18 +68,16 @@ const userSchema = z.object({
 @Router()
 class UserRouter {
   constructor(
-    @Inject(UserService) private readonly userService: UserService
```

Comment:
I'd rather move this to the top and keep the Idiomatic way of the `@Inject()` decorator to emphasis that DI works seamlessly with this library.